### PR TITLE
Update backup with facebooks

### PR DESCRIPTION
**Pull request by stephengroat**

### CODE REVIEW

**Review by fpigerre — APPROVED** [review]

File: _data/backup.yml [hunk 1]
```diff
@@ -41,12 +41,14 @@ websites:
     twitter: cloudapp
     img: cloudapp.png
     tfa: No
+    facebook: getcloudapp
 
   - name: CrashPlan
     url: http://www.crashplan.com
```
Comment (Jawshy) [inline _data/backup.yml]: They are using HTTPS now. @stephengroat, would you like update the URL here or should it be done in a separate pull request?

**Comment by Jawshy** [comment]
Second `looks-good`[†](https://github.com/2factorauth/twofactorauth/pull/2010#discussion_r83551307) unless we are going to be strict enough to have the `twitter` and `facebook` fields ordered directly above/below each other.
